sync/block_listener.go

```diff
@@ -87,7 +87,7 @@ func (bl *BlockListener) handleBlock(data service.GossipMessage) {
 	//set the block id when received
 	blk.Initialize()
 
-	bl.Log.With().Info("block received",
+	bl.Log.With().Info("got new block",
```

Comment:
You forgot to update the matching lines in the test file here:
https://github.com/spacemeshos/go-spacemesh/blob/6f4af8b0f1021ea7a9b56349b4ffc1333dbc5b60/tests/queries.py#L440

This test doesn't currently run for an unrelated reason (someone accidentally[?] removed it from `test_bs`), so I don't blame you, but we should still fix this.

In general, I consider @barakshani's code review a bonus, but wouldn't merge any code without review from a "real dev" 😉

Reply:
you cannot forget something you dont know, i consider non running tests the issue of the person who skiped them, you cannot support code that you don't know about and is not being used.

Reply:
🤦‍♂

Reply:
🙅🏻‍♂️